README.md

```diff
@@ -2326,11 +2327,166 @@ Upgrading the Resource Aggregator for ODIM deployment involves:
 
     
 
+## Backup and restore of Redis
 
+1. Create a directory to store the backup rdb files and append-only files.
 
+   ```
+   mkdir backup
+   ```
+
+2. Get the name of the master pod to collect the backup snapshot file and appendonly file to restore them later (either ondisk or inmemory database).
+
+   ```
+   kubectl get pods -nodim | grep redis | grep primary
+   ```
+   
+   **Output**:
+
+   ```
+   redis-ha-inmemory-primary-0
+   ```
+   ```
+   redis-ha-ondisk-primary-0
+   ```
+
+3. Copy the rdb files and aof files from the master pod.
+
+   ```
+   kubectl cp odim/<pod-name>:/redis-data/dump.rdb /home/bruce/backups/dump.rdb
+   ```
+
+   ```
+   kubectl cp odim/<pod-name>:/redis-data/appendonly.aof /home/bruce/backups/appendonly.aof
+   ```
 
+   **For example (on-disk)**:
+   
+   ```
+kubectl cp odim/redis-ha-ondisk-primary-0:/redis-data/dump.rdb /home/bruce/backups/dump.rdb
+   ```
 
+   ```
+   kubectl cp odim/redis-ha-ondisk-primary-0:/redis-data/appendonly.aof /home/bruce/backups/appendonly.aof
+   ```
+
+   **For example(in-memory)**:
+   
+   ```
+kubectl cp odim/redis-ha-inmemory-primary-0:/redis-data/dump.rdb /home/bruce/backups/dump.rdb
+   ```
+   
+   ```
+kubectl cp odim/redis-ha-inmemory-primary-0:/redis-data/appendonly.aof /home/bruce/backups/appendonly.aof
+   ```
+   
+   These are the backup files to be restored.
+
+4. Copy the above backup files to the pod to restore them.
+
+   ```
+   kubectl cp ./backups/dump.rdb odim/<pod-name>:/redis-data/dump.rdb-1
+   ```
+
+   ```
+   kubectl cp ./backups/appendonly.aof  odim/pod-name>:/redis-data/appendonly.aof.old
+   ```
+
+   **For example**:
+
+   ```
+   kubectl cp ./backups/dump.rdb odim/redis-ha-ondisk-primary-0:/redis-data/dump.rdb-1
+   ```
+
+   ```
+   kubectl cp ./backups/appendonly.aof  odim/redis-ha-ondisk-primary-0/redis-data/appendonly.aof.old
+   ```
+
+   ```
+   kubectl cp ./backups/dump.rdb odim/redis-ha-inmemory-primary-0:/redis-data/dump.rdb-1
+   ```
+
+   ```
+   kubectl cp ./backups/appendonly.aof  odim/redis-ha-inmemory-primary-0/redis-data/appendonly.aof.old
+   ```
+
+5. Log in to redis CLI and disable append-only configuration to restore the data back.
```

Comment:
Find out from Shahid, from where this redis-cli has to be executed?

Reply:
fixed